cost/google/schedule_instance/google_schedule_instance.pt

```diff
@@ -325,16 +325,7 @@ define schedule_instance($data) return $all_response do
     end
 	call get_tzlist() retrieve $timezones
 	$timezones=from_json($timezones)
-	$count=0
-    foreach $obj in $timezones do
-      if $count==0
-        $var=to_a($obj)
-        if $var[0][0]==$timezone
-          $timezone=$var[0][1]
-          $count=$count+1
-        end
-      end
-    end
+	$timezone=$timezones[$timezone]
```

Comment:
what happens if the $timezone isn't typed correctly or isn't found?

Reply:
I have added the code to handle this. If the timezone is not found, sys log will show that error. Thanks